app/serializers/logs_serializer.rb

```diff
@@ -10,7 +10,7 @@ def user_id
   end
 
   def username
-    User.find_by(id: object.whodunnit).username
+    User.find_by(id: object.whodunnit.to_i).username
```

Comment:
`User.find(object.whodunnit).username`
does the exact same thing :) 
`.find()` is the default by id search.